File: 2609.py
a, b = map(int, input().split())

# 시간초과
# for i in range(min(a, b), 0 , -1) :
#     if a % i == 0 and b % i == 0:
#         print(i)

# for i in range(max(a, b), (a * b) + 1) :
#     if i % a == 0 and i % b == 0:
#         print(i)

def GCD(a, b):
    while (b):
        a, b = b, a % b
    return a

def LCM(a, b):
    result = (a * b) // GCD(a, b)
    return result

print(GCD(a, b))
print(LCM(a, b))

File: 201902646/2609.cpp
#include <iostream>

int gcd(int a, int b) {
    int n = a % b;
    while(n != 0) {
        a = b;
        b = n;
        n = a % b;
    }
    return b;
}

int lcm(int a, int b) {
    return (a * b) / gcd(a,b);
}

int main() { 
    int a, b;
    std::cin >> a >> b;
    std::cout << gcd(a,b) << std::endl;
    std::cout << lcm(a,b) << std::endl;
}
